draft-ietf-core-cf-reg-update.md

```diff
@@ -37,6 +37,7 @@ author:
 normative:
   RFC7120: iana-early
   RFC7252: coap
+  RFC9110:
```

Comment:
Suggest aliasing this to `http-sema` and, in the refs,  using that label instead of the raw RFC number.